common/lib/xmodule/xmodule/course_module.py

```diff
@@ -264,18 +267,32 @@ class CourseFields(object):
     )
     discussion_blackouts = List(
         display_name=_("Discussion Blackout Dates"),
-        help=_("Enter pairs of dates between which students cannot post to discussion forums. Each pair should be formatted as [\"YYYY-MM-DD\", \"YYYY-MM-DD\"]. To specify times as well as dates, format each pair as [\"YYYY-MM-DDTHH:MM\", \"YYYY-MM-DDTHH:MM\"] (be sure to include the \"T\" between the date and time). An entry defining more than one blackout period might look like this: [[\"2014-09-15\", \"2014-09-21\"], [\"2014-10-01\", \"2014-10-08\"]]"),
+        help=_(
+            "Enter pairs of dates between which students cannot post to discussion forums. Each pair should be "
+            "formatted as [\"YYYY-MM-DD\", \"YYYY-MM-DD\"]. To specify times as well as dates, format each pair "
+            "as [\"YYYY-MM-DDTHH:MM\", \"YYYY-MM-DDTHH:MM\"] (be sure to include the \"T\" between the date and "
+            "time). An entry defining more than one blackout period might look like this: "
+            "[[\"2014-09-15\", \"2014-09-21\"], [\"2014-10-01\", \"2014-10-08\"]]"
```

Comment:
Hmm. So in Transifex, these characters do NOT show up escaped, so the translations do NOT specify the characters with the escape notation. Is this problematic?

![screen shot 2015-01-21 at 5 27 54 pm](https://cloud.githubusercontent.com/assets/1985317/5846604/e939653a-a192-11e4-9e63-9d609e0484f3.png)

Sorry Giulio - I know this doesn't have anything to do with this change - but since it's been raised, we should make it right.